src/clr-addons/notification/notification.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../../node_modules/@clr/ui/src/layout/grid/variables/variables.clarity';
```

Comment:
you (most likely, please test) dont have to import this to access clr variables in this repo - we import all of them automatically and use/override them.